echo.js

```diff
@@ -1,5 +1,6 @@
 function echo(){
-  // ...
+  for (var i=0; i<arguments.length; i++) 
+        console.log(arguments[i]);
```

Comment:
I'd recommend always using curly braces around the bodies of `for` loops, even when they aren't required.